### PR TITLE
Update unified dashboard tab bar pills to scroll to an area instead of relying on the browser anchor targets.

### PR DESCRIPTION
**Pull request by techanvil**
## Summary

Addresses issue #4367 

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/components/DashboardNavigation.js [hunk 1]
```diff
@@ -37,13 +38,49 @@ import {
...
 
+/**
+ * Get the y coordinate to scroll to the top of a context element, taking the sticky header and navigation height into account.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} contextID The ID of the context element to scroll to.
+ * @return {number} The offset to scroll to.
+ */
+
```
Comment (eugene-manuilov) [inline assets/js/components/DashboardNavigation.js:49]:
```suggestion
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
LGTM. Added one suggestion to remove an empty line. I'll apply it by myself.

**Review by eugene-manuilov — APPROVED** [review]

**Comment by techanvil** [comment]
> LGTM. Added one suggestion to remove an empty line. I'll apply it by myself.

Thanks @eugene-manuilov, I must try to remember not to leave an empty line between JSDoc and the function it describes.

Having removed that extra line I can see the linting has kicked in for the JSDoc - I'll push a commit to fix the error.

**Review by eugene-manuilov — APPROVED** [review]
Thanks, @techanvil. LGTM!

**Pull request merged**